R/replaceLinkingError.r

```diff
@@ -3,6 +3,8 @@
 replaceLinkingError <- function(equatingList, multiEquatError_output, verbose = TRUE, digits = 4) {
     ### checks
        if ( !is.list(multiEquatError_output)&&is.null(dim(multiEquatError_output)) ) {stop("'multiEquatError_output' must be a list of data.frames.")}

```

Comment:
Should I leave existing tests as they are or try to rewrite them with checkmate?

Reply:
If there is a syntactically simpler (or at least not more complex) way to implement these tests with checkmate, feel free! Thanks!